Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1898,13 +1908,22 @@ void CGContextSetPatternPhase(CGContextRef context, CGSize phase) {
 #pragma endregion
 
 #pragma region Drawing Operations - Text
+
+// Attributes are needed in DrawGlyphRuns but names are defined in CoreText
```

Comment:
I don't know that it would make sense to define these in CG and export them for CT (especially since we're not sharing values w/ UIKit consts).  Not stubborn on this so if others think this is worth changing I'll fix it.